DirectX11/HackerContext.h

```diff
@@ -9,6 +9,7 @@
 
 #include "HackerDevice.h"
 //#include "ResourceHash.h"
+#include "..\HackingCommon\globals_common.h"
```

Comment:
There's a couple of other places you removed or added a "..". I'll never object to these being removed, but if you're adding one it probably indicates that the include path isn't set correctly for the project and that should be fixed instead.